database.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE IF NOT EXISTS `Database shizzle`;
+
+CREATE TABLE `table enzo `(
+	ID      INT              NOT NULL AUTO_INCREMENT,
+  	Author  VARCHAR (20)    ,
```

Comment:
20 characters is maybe to lil for a name.

Reply:
I think you are right will check it out thnx

Reply:
As Filip said now, no limit is necessary actually.